### PR TITLE
Update community page

### PR DESCRIPTION
**Pull request by eliasmbd**
Here is the updated community layout. 

Still need to review links

### CODE REVIEW

**Comment by eliasmbd** [comment]
Replaces #15

File: docs/community/get_involved.md [hunk 1]
```diff
@@ -7,28 +9,76 @@
 
 </div>
 
+## GTFS Governance
+
+The GTFS Specification is not set in stone. Instead, it is an open specification developed and maintained by the community of transit agencies, developers, and other stakeholders who use GTFS. It is expected that this community of producers and consumers of GTFS data will have proposals for extending the spec to enable new capabilities.
+
+To formalize this process, we've established the GTFS Governance framework, which includes the Specification Amendment Process (SAP).
```
Comment (Sergiodero) [inline docs/community/get_involved.md]:
```suggestion
To formalize this process, we've established the GTFS Governance framework, which includes the Specification Amendment Process.
```

File: docs/community/get_involved.md [hunk 2]
```diff
@@ -7,28 +9,76 @@
...
+
+<div class="grid cards" markdown>
+
+-   :material-file-document-edit:{ .lg .middle } __GTFS Schedule - Specification Amendment Process__
+
+    [:octicons-arrow-right-24: Learn more](../../resource_library/spec_amendment_process/gtfs_schedule_amendment_process)
```
Comment (Sergiodero) [inline docs/community/get_involved.md:22]: We need to update this link once PR #30 is merged, you can ignore this comment when merging, it's just a reminder

File: docs/community/get_involved.md [hunk 3]
```diff
@@ -7,28 +9,76 @@
...
+
+-   :material-file-document-edit:{ .lg .middle } __GTFS Realtime - Specification Amendment Process__
+
+    [:octicons-arrow-right-24: Learn more](../../resource_library/spec_amendment_process/gtfs_realtime_amendment_process)
```
Comment (Sergiodero) [inline docs/community/get_involved.md:26]: Same as avobe: update this link once PR #30 is merged

File: docs/community/get_involved.md [hunk 4]
```diff
@@ -7,28 +9,76 @@
 
 </div>
 
+## GTFS Governance
+
+The GTFS Specification is not set in stone. Instead, it is an open specification developed and maintained by the community of transit agencies, developers, and other stakeholders who use GTFS. It is expected that this community of producers and consumers of GTFS data will have proposals for extending the spec to enable new capabilities.
```
Comment (Sergiodero) [inline docs/community/get_involved.md:14]: This text is included in each Spec Amendment Process, should we remove it from there and keep it in here?

**Comment by Sergiodero** [comment]
Thanks for working on this! The page is displaying correctly and it's looking good, I only left a [suggestion](https://github.com/MobilityData/elrond.gtfs.org/pull/27#discussion_r1603991786) to cut the SAP acronym. 

We still need to review all links, so we can either merge as is and open a new PR that only focuses on this, or wait for other PRs to be merge since this is dependant on them (PR #30 and PR #29).

I'm okay with both,  so you can decide. LMK and I can approve afterwards. Thanks!

File: docs/community/get_involved.md (hunk 4 above)
Comment (eliasmbd) [reply docs/community/get_involved.md:14]: id keep it

**Comment by eliasmbd** [comment]
> Thanks for working on this! The page is displaying correctly and it's looking good, I only left a [suggestion](https://github.com/MobilityData/elrond.gtfs.org/pull/27#discussion_r1603991786) to cut the SAP acronym.
> 
> We still need to review all links, so we can either merge as is and open a new PR that only focuses on this, or wait for other PRs to be merge since this is dependant on them (PR #30 and PR #29).
> 
> I'm okay with both, so you can decide. LMK and I can approve afterwards. Thanks!

Let's merge this one and then create other PR for the subsequent steps.

**Review by Sergiodero — APPROVED** [review]
Perfect, I'm approving! Thanks again!

**Pull request merged**